docs/COC.md

```diff
@@ -0,0 +1,55 @@
+# Code of Conduct
+
+This code of conduct outlines expectations of the Open Biological Ontologies (OBO) community. The OBO community is committed to providing a welcoming and inspiring environment for all community members, and we expect this code of conduct to be honored by each community member. Herein, we provide specifics regarding appropriate code of conduct, including recommendations for conflict resolution. The OBO community strives to: 
+
+**Be welcoming and patient** 
+
+We want and need all-comers to contribute to our collective goals of creating a robust suite of interoperable and effective ontologies. Cooperation from all members is needed to help ensure a safe and welcoming environment for everybody. We strive to be a community that that welcomes and supports persons of all backgrounds and identities. This includes, but is not limited to, members of any race, ethnicity, culture, national origin, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, physical size, family status, political belief, religion, and mental or physical ability.
+
+**Be considerate** 
+
+Your work will be used by other people, and you will depend on the work of others. Any decision you take will affect other community members. The consequences of your actions should be considered when making decisions. Remember that we're a large community, so you might not be communicating in someone else's primary language or share their area of expertise. This can present a challenge, but with thoughtful consideration, all communications between community members can be, and should be, considerate of the perspectives of others.
+
+**Be respectful** 
+
+Community members are likely to disagree on issues. This is to be expected (especially amongst ontologists!),  within our large, diverse group of individuals. However, we aim to support, at all times, constructive discourse when disagreements arise. This may lead to frustration from time to time, but we cannot allow frustration to turn into disrespectful behavior, mannerisms, or attacks. Our collective productivity very much depends on a supportive community, where people feel comfortable or not threatened when they need to disagree or propose alternative strategies.
```

Comment:
This is key! We robustly disagree on the very nature of reality, but we're pretty cool about it and can still have beer (or a non-alcoholic beverage) with each other afterwards.